### PR TITLE
Travel Website using React JS

### LINKED ISSUE #21: Travel Website using React JS
### WebMasterLog Repository (Proposing new issue)
:red_circle: **Project Title** : A Travel Website
:red_circle: **Aim** : A website to showcase the different places of the world to the tourist to go there for a tour.
:red_circle: **Tech Stack Used** : React
:red_circle: **Category** :  React-JS

***********************************************************************
### 📍 **Follow the Guidelines to Contribute in the Project :**
* You need to create a separate folder named as the Project Title.
* Inside that folder, there will be all the main components.
   * **Cover image** of the application, if project will be deployed in future, with the name of the folder in Kebab-Case (hypen-separated).
   * Main files in formatted state so that code will be readable.
   * Other Images - In seprate folder named `Images` used to form the application.
* Inside the `Project` folder, the `README.md` file must be filled up properly, with proper visualizations and descriptions.

***********************************************************************
:red_circle::yellow_circle: **Points to Note :**

- The issues will be assigned on a first come first serve basis, 1 Issue == 1 PR.
- "Issue Title" and "PR Title should be the same. Include issue number along with it.
- Follow [Contributing Guidelines](https://github.com/Avdhesh-Varshney/WebMasterLog/blob/main/CONTRIBUTING.md) & [Code of Conduct](https://github.com/Avdhesh-Varshney/WebMasterLog/blob/main/CODE_OF_CONDUCT.md) before start Contributing.

***********************************************************************
:white_check_mark: **To be Mentioned while taking the issue :**
- Full name : 
- GitHub Profile Link : 
- What is your participant role? (Mention the Open Source Program name. Eg. GSOC, GSSOC, SSOC, JWOC, etc.)

***********************************************************************
Happy Contributing 🚀 

All the best. Enjoy your open source journey ahead. 😎

### PR DESCRIPTION
**Pull request by aditya-bhaumik**
Closes #21 

- **Title** :  Travel Website
- **Name:** : Aditya Bhaumik
- **Idenitfy yourself:** : GSSOC'24

### Describe the add-ons or changes you've made 📃

I have added travel website in React-Projects>Intermediate and also updated database , readme files

### How Has This Been Tested? ⚙️

What sort of change have you made:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### Checklist: ☑️

- [x] My code follows the [Contributing Guidelines](https://github.com/Avdhesh-Varshney/WebMasterLog/blob/main/CONTRIBUTING.md) & [Code of Conduct](https://github.com/Avdhesh-Varshney/WebMasterLog/blob/main/CODE_OF_CONDUCT.md) of this project.
- [x] This PR does not contain plagiarized content.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly wherever it was hard to understand.
- [x] My changes generate no new warnings.

### Note to reviewers 📄
Fixes #21 

Sorry it took some time to make this website
Please review my pull request and add the gssoc label 
Thank you

### CODE REVIEW

**Comment by aditya-bhaumik** [comment]
image

File: React-JS-Projects/Advanced/Travel-Website/README.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+
+![Line](https://github.com/Avdhesh-Varshney/WebMasterLog/assets/114330097/4b78510f-a941-45f8-a9d5-80ed0705e847)
+
+<!-- -------------------------------------------------------------------------------------------------------------- -->
+
+<h4 align='center'>Developed By <b><i>Aditya Bhaumik</i></b> 👦</h4>
+<p align='center'>
+    <img src='https://img.shields.io/badge/linkedin-%230077B5.svg?style=for-the-badge&logo=linkedin&logoColor=white' />
+    <img src='https://img.shields.io/badge/github-%23121011.svg?style=for-the-badge&logo=github&logoColor=white' />
```
Comment (Avdhesh-Varshney) [inline React-JS-Projects/Advanced/Travel-Website/README.md:63]: Mention your linkedin and github profile properly not just the tags.

File: React-JS-Projects/Advanced/Travel-Website/public/screenshot.webp
Comment (Avdhesh-Varshney) [inline React-JS-Projects/Advanced/Travel-Website/public/screenshot.webp:1]: keep it outside the public folder

**Review by Avdhesh-Varshney — CHANGES_REQUESTED** [review]

**Comment by Avdhesh-Varshney** [comment]
@aditya-bhaumik Also attach your assigned issue with the pr by editing the pr template

**Comment by aditya-bhaumik** [comment]
@Avdhesh-Varshney i have made the changes requested

**Review by Avdhesh-Varshney — APPROVED** [review]
@aditya-bhaumik PR Approved 🎉

**Comment by Avdhesh-Varshney** [comment]
@aditya-bhaumik don't create a large sequence of commits for a single pr.

**Comment by aditya-bhaumik** [comment]
@Avdhesh-Varshney i am really sorry i will make sure not to do this mistake in the next project actually i had completed the project but for some reason i was not able to push the project using my terminal or git bash

**Pull request merged**